pages/es/learningcenter/mellon-on-the-markets.html

```diff
@@ -13,6 +13,14 @@
 		</div>
 		<div class="Page-Grid__grid-wrap">
 		    <div class="Page-Grid__grid-wrap__box">
+				<a href="{{site.url}}/ebooks/ebook_en_MellonOnTheMarketSep2016.pdf">
```

Comment:
The point is to use URLs which can be resolved in both production and travis as well.

The current travis error is because `htmlproofer` returns _404 Not Found_ error for URLs which can't be found at the moment. (Using `site.url`, it tries to find the file on `https://academy.binary.com`, but it's not there yet before merging this PR.)

In this case the solution would be `/ebooks/ebook_en_MellonOnTheMarketSep2016.pdf`.

`site.baseurl` seems to be a common practice, because in case of having a subpath, it'll be added to your URLs. It's empty for now, so the URL will resolve to the above root relative that can be accessed by travis through this PR.